ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -814,72 +590,53 @@ public void shouldBuildTableWithCorrectStoreName() {
     transformer.init(processorCtx);
     return transformer;
   }
-
-  private void givenWindowedSourceStream() {
+  private void givenWindowedSourceTable(final int pseudoColumnVersion) {
     when(buildContext.buildKeySerde(any(), any(), any(), any())).thenReturn(windowedKeySerde);
     givenConsumed(consumedWindowed, windowedKeySerde);
-    windowedStreamSource = new WindowedStreamSource(
+    givenConsumed(consumedWindowed, windowedKeySerde);
+    windowedTableSource = new WindowedTableSource(
         new ExecutionStepPropertiesV1(ctx),
         TOPIC_NAME,
         Formats.of(keyFormatInfo, valueFormatInfo, KEY_FEATURES, VALUE_FEATURES),
         windowInfo,
         TIMESTAMP_COLUMN,
         SOURCE_SCHEMA,
-        OptionalInt.of(SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER)
+        OptionalInt.of(pseudoColumnVersion)
     );
   }
 
-  private void givenUnwindowedSourceStream() {
-    when(buildContext.buildKeySerde(any(), any(), any())).thenReturn(keySerde);
-    givenConsumed(consumed, keySerde);
-    streamSource = new StreamSource(
-        new ExecutionStepPropertiesV1(ctx),
-        TOPIC_NAME,
-        Formats.of(keyFormatInfo, valueFormatInfo, KEY_FEATURES, VALUE_FEATURES),
-        TIMESTAMP_COLUMN,
-        SOURCE_SCHEMA,
-        OptionalInt.of(SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER)
-    );
+  private void givenWindowedSourceTable() {
+    givenWindowedSourceTable(SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER);
   }
 
-  private void givenMultiColumnSourceStream() {
+  private void givenUnwindowedSourceTable(final int pseudoColumnVersion) {
```

Comment:
No point in having this overload (to accept a `pseudoColumnVersion`) if all the tests use `CURRENT_PSEUDOCOLUMN_VERSION_NUMBER` anyway. Same for `givenWindowedSourceTable()` above.